### PR TITLE
Prepare Netlify hosting

### PR DESCRIPTION
**Pull request by uncenteredDiv**
I noticed an issue with the `build` command - the `static-dirs` path was wrong - due to this the task always failed.
Since the `decks/assets` folder needs to be in the Git, I moved one picture from my previous talk there.
With my next Talk there will be also more images there.

I also added the `build` folder to the .gitignore and updated the sorting / removed some unneeded files.
We might generate a .gitignore with [gitignore.io](https://www.toptal.com/developers/gitignore) next time - but IMO it's too much.

Out of habit, I added the WebStorm Run Configs for the `start`, `build` and `serve` commands ;)

### CODE REVIEW

**Comment by uncenteredDiv** [comment]
I somehow fucked up my commit user 🤷‍♂️ 
But it should be fixed now 👍 

Btw: I setup a Netlify Deploy for my forked repo with the changes from this PR -> https://determined-mccarthy-de74e8.netlify.app/

File: .gitignore [hunk 1]
```diff
@@ -1,13 +1,14 @@
+# WebStorm
 .idea/
-.history/
```
Comment (orioltf) [inline .gitignore:2]: I would bring this one back. It's a known VSCode plugin that keeps local history changes.

Comment (uncenteredDiv) [reply .gitignore:2]: We could also use this template
https://www.toptal.com/developers/gitignore/api/webstorm,vscode,react

Comment (orioltf) [reply .gitignore:2]: Looks good to me!

Comment (orioltf) [reply .gitignore:2]: I'd make a new PR for that, however, so that we can merge the main topic of this PR already. Fine for you? If so, feel free to merge

**Pull request merged**